actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -23,9 +25,9 @@ class ActionCable.Connection
       ActionCable.log("Attemped to open WebSocket, but existing socket is #{@getState()}")
       throw new Error("Existing connection must be closed before opening")
     else
-      ActionCable.log("Opening WebSocket, current state is #{@getState()}")
+      ActionCable.log("Opening WebSocket, current state is #{@getState()}, subprotocols: " + protocols.join(","))
```

Comment:
Arrays are implicitly joined with `,` when coerced to a String. Can do `.log("Opening WebSocket, current state is #{@getState()}, subprotocols: #{protocols}")`